docs/sphinx/source/whatsnew/v0.6.0.rst

```diff
@@ -5,13 +5,14 @@ v0.6.0 (___, 2018)
 
 API Changes
 ~~~~~~~~~~~
-
+* pvsystem.calcparams_desoto now accepts only keyword arguments for each module model parameter. Prior to v0.6.0, pvsystem.calcparams_desoto required some module model parameters as keyword argument and accpted a dict, module_parameters, as an input providing the remaining module model parameters.
```

Comment:
Recommend something more like:

pvsystem.calcparams_desoto now requires arguments for each module model parameter.

The arguments you've added are positional, not [keyword-only](https://www.python.org/dev/peps/pep-3102/).

Reply:
The whatsnew files generally use an 80 character line limit, though this is not as strict as for python modules.

Reply:
accpted --> accepted

Reply:
shortened to `pvsystem.calcparams_desoto now requires arguments for each module model parameter.` to stay within character limits.

Reply:
Ok. To be clear and for future reference, you can also continue on to additional lines if you indent them by 2 spaces so that it looks like

```
* this is a long item....
  that continues here...
```